htdocs/product/class/product.class.php

```diff
@@ -5080,6 +5080,9 @@ public function getTooltipContentArray($params)
 		if (!empty($this->label)) {
 			$datas['label']= '<br><b>'.$langs->trans('ProductLabel').':</b> '.$this->label;
 		}
+		if (!empty($this->description)) {
+			$datas['description']= '<br><b>'.$langs->trans('ProductDescription').':</b> '.$this->description;
```

Comment:
What happen if product description is several page of HTML content ?
I suggest to add dolGetFirstLineOfText() around text to keep only the 5 first lines...

Reply:
Yes, that is indeed helpful.
I will modify that.